### ISSUE TITLE
Enable `scratch-notifier` per device

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#6864]
### Why this would be helpful

Some users might want to have notifications come in on some devices and not others, but since Scratch Addons' settings sync across devices, this isn't possible unless you're using a browser profile that doesn't sync settings.

This could become more useful once we start supporting mobile devices, where people may be running Scratch Addons on their tablets or even phones for the notifications.

### My suggestion

Don't sync the state of Scratch Notifier across devices. Instead, whether it is enabled could be stored separately in local storage, or it could be determined by whether you have allowed Scratch Addons to display notifications (related: #6863).

### Additional context

Might also be a good time to address #6401, the issue where the addon could get disabled if you use it on a device where Scratch Addons does not have permission to display notifications.

**Comment by WorldLanguages** [comment]
Yes, it is true that even if we ignore or solve the sync issues related to the optional permissions, users might still want to have this enabled or disabled based on which device they're using.
Theoretically we could set up a very complicated system to extend this to other addons and settings, but realistically it's not going to happen.

**Comment by WorldLanguages** [comment]
Somewhat related to #5431
